backend/habragasolina/src/main/java/bo/edu/usfa/gasolina/habragasolina/Service/GasStationService.java

```diff
@@ -0,0 +1,32 @@
+package bo.edu.usfa.gasolina.habragasolina.Service;
+
+import bo.edu.usfa.gasolina.habragasolina.Entities.GasStationEntity;
+import bo.edu.usfa.gasolina.habragasolina.Repository.GasStationRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class GasStationService {
+
+    private final GasStationRepository gasStationRepository;
+
+    @Autowired
+    public GasStationService(GasStationRepository gasStationRepository) {
+        this.gasStationRepository = gasStationRepository;
+    }
+
+    public GasStationEntity saveGasStation(GasStationEntity gasStation) {
+        return gasStationRepository.save(gasStation);
```

Comment:
There has to be a validation for the name, because it has to be at least three characters